### PR TITLE
react project finish

### PR DESCRIPTION
**Pull request by digitalAsh**

### CODE REVIEW

File: Project-I-Social-Card/social-card/src/App.js [hunk 1]
```diff
@@ -1,23 +1,17 @@
...
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="App">
+      <HeaderContainer />
+      <a href='https://reactjs.org/' style={{ textDecoration: 'none' }}><CardContainer /></a>
```
Comment (ielvisd) [inline Project-I-Social-Card/social-card/src/App.js:11]: Nice use of the anchor tag to direct to the React website.

**Review by ielvisd — APPROVED** [review]
You might want to add some more padding as things look a bit squished at the moment but that's minor. Great work getting the card to look similar to the design file.

**Comment by ielvisd** [comment]
Hey Ashley, don't forget to do a separate pull request for different projects. No big deal this time, your calculator project looks good. One thing I would explore is how to render each button in one go rather than hard coding in each one like we did in lecture.

**Comment by digitalAsh** [comment]
Thanks for the feedback.  Its hard for me to do these projects different
than what we learn in the lecture but I think the more i practice the
better i will get at improvising.

**Pull request merged**